README.md

```diff
@@ -32,9 +32,3 @@ Let us know how experimenting with this demo goes! Feel free to [open an Issue](
 ## What's next
 
 To get to know more Daily API methods and events, explore our other demos, like [how to add your own chat interface](https://github.com/daily-co/daily-demos/tree/main/static-demos/simple-chat-demo).
-
----
-
-## Related blog posts/tutorials
```

Comment:
I think we should leave this link in. it's a great starting point. was there a reason to remove it?

Reply:
oh wait. this was the react native link. for some reason even after clicking it i thought it was the daily-x intro 🤦‍♀️